kolibri/plugins/learn/assets/test/util/SlottedDiv.vue

```diff
@@ -2,7 +2,7 @@
 
   <div>
     <div>Hey I'm Empty Inside</div>
-    <slot />
+    <slot/>
```

Comment:
Does this need to be `<slot></slot>` too like everywhere else?

Reply:
Yeah, it's getting flagged in my editor, but I don't think test webpack config runs eslint, which is why it probably passed.

Reply:
Looks like this file isn't being linted. Since it is not referenced from any other place?

Reply:
It's a mock used in some tests, and the tests themselves don't run ESLint.